great_expectations/data_context/data_context/base_data_context.py

```diff
@@ -181,38 +185,48 @@ def __init__(
                 based on conventions for project subdirectories.
             runtime_environment: a dictionary of config variables that
                 override both those set in config_variables.yml and the environment
-            ge_cloud_mode: boolean flag that describe whether DataContext is being instantiated by ge_cloud
-           ge_cloud_config: config for ge_cloud
+            cloud_mode: boolean flag that describe whether DataContext is being instantiated by Cloud
+            cloud_config: config for Cloud
         Returns:
             None
         """
+        # If any deprecated args deviate from their defaults, we should flag the issue
+        if ge_cloud_mode is not False or ge_cloud_config is not None:
```

Comment:
Is this equivalent to `if ge_cloud_mode`?

Reply:
hmm probably since you won't be passing in `ge_cloud_config` without `ge_cloud_mode` but I added the conditional for all input arguments (even if one relies on the presence of another).

Reply:
so `ge_cloud_mode` is weird here because it is an `Optional[bool]` - the other values can be truthy/falsy checks but our Cloud mode treatment is particular and requires us to check if there's a deviation from the default `False` (as opposed to `None`).

Reply:
if this code path is called by cloud snippets, we should defer the deprecation warning until after the snippets are updated

Reply:
Just noticed that there were two variables here. This makes sense. 👍🏻